src/LibHdb++Cassandra.cpp

```diff
@@ -1417,33 +510,21 @@ void HdbPPCassandra::insert_Attr(Tango::EventData *data, HdbEventDataType ev_dat
     }
     else
     {
-        attr_r_dim.dim_x = 0; // max_dim_x;//TODO: OK?
-        attr_w_dim.dim_x = 0; // max_dim_x;//TODO: OK?
-        attr_r_dim.dim_y = 0; // max_dim_y;//TODO: OK?
-        attr_w_dim.dim_y = 0; // max_dim_y;//TODO: OK?
+        /// @todo Check this is ok?
```

Comment:
The dimensions seem to be never set in the Cassandra tables since we are currently using a CQL list to store TANGO spectrum attributes data and the length of this list is used directly by the java HDB++ extraction library to get the dimensions (for read and write value) of the spectrum attribute. So the attr_r_dim and attr_w_dim variables seem to be useless in the current implementation. The dim_x and dim_y columns in att_array__tango_type_rw||ro tables are also currently useless (not your fault of course).

Reply:
Perhaps we should just remove them for now?

Reply:
It sounds reasonable to remove them, even from the CQL tables. We should do some tests before to do it on the real system though to ensure there is no side effect but at first sight, it seems it should be fine but I might have missed something...

Reply:
They do not appear in any tables. Removed the variables and I think that is it.